### DISCUSSION TITLE
Where to update list of `origin/domain` addresses for `verify.walletconnet.org` endpoint `CSP` ?

### DISCUSSION THREAD

**Discussion by migbash** [reown-com/appkit#2988] [Q&A]
I need some help with the use of the https://cloud.reown.com/ for changing the `origin/domain` available for a project.

This is because of an error that keeps appearing on the `devtools`:

<img width="412" alt="image" src="https://github.com/user-attachments/assets/20392d8e-5173-4eb9-b592-9db49d2f3147">

**Comment by rtomas** [comment]
whats your problem ? you cannot change your domain ?

**Reply by migbash** [reply]
Yes, I am getting some errors that seem to be regarding invalid `IP/domain` configuration.

**Reply by rtomas** [reply]
do you have an online web ?

**Reply by migbash** [reply]
Yes

**Reply by rtomas** [reply]
Did you add it to the whitelist?
can you share the url ?

**Reply by javaspeak** [reply]
How do we add it to a whitelist and which whitelist do we add it to?

I thought that if we verify the domain that our domain will be added to the CSP headers by reown?

**Comment by x0good** [comment]
@migbash maybe you need check your CSP policy https://docs.reown.com/advanced/security/content-security-policy on your side (as I see you upload screen from dev tools)?

check `frame-src`

**Reply by javaspeak** [reply]
My domain has been validated  but I still get this error when trying to login to AppKit using the Google option:

Refused to frame 'https://secure.walletconnect.org/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'self' http://localhost:* https://*.pages.dev https://*.vercel.app https://*.ngrok-free.app https://secure-mobile.walletconnect.com https://secure-mobile.walletconnect.org".

Are we supposed to do some CSP config as well?

**Comment by ghost** [comment]
@rtomas hello, we use vercel for development env, how to add `*.vercel.app` to the verified domains? since it's a domain of vercel, we don't have control
